cli/js/web/custom_event.ts

```diff
@@ -5,18 +5,12 @@ import { requiredArguments } from "./util.ts";
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export class CustomEventImpl<T = any> extends Event implements CustomEvent {
-  readonly #detail: T;
+  readonly detail: T;
```

Comment:
Please revert.  `readonly` is a TypeScript only construct and to be spec compliant it needs to be truly read-only.